include/TMS/dev/TCA9545A.h

```diff
@@ -25,15 +25,17 @@ enum TCA9545A_BUS {
  * TCA9545A I2C Multiplexer Driver
  * https://www.ti.com/lit/ds/symlink/tca9545a.pdf
  */
-class TCA9545A : I2CDevice::I2CDevice {
+class TCA9545A {
 public:
 
     /**
      * Constructor for the TCA9545A driver
      *
-     * @param i2c I2C device for communicating with TCA
+     * @param[in] i2c I2C device for communicating with TCA
+     * @param[in] addr address of I2C device
+     * @param[in] buses array of buses containing I2CDevices
      */
-    TCA9545A(IO::I2C& i2c, uint8_t addr = TCA9545A_DEFAULT_I2C_ADDR);
+    TCA9545A(IO::I2C& i2c, uint8_t addr = TCA9545A_DEFAULT_I2C_ADDR, I2CDevice** buses[4] = nullptr);
```

Comment:
Why are you setting the buses to be `nullptr` by default? This parameter probably shouldn't be optional